discord/ext/commands/context.py

```diff
@@ -206,6 +210,21 @@ def valid(self):
     async def _get_channel(self):
         return self.channel
 
+    @property
+    def clean_prefix(self):
+        """clean_prefix: :class:`str`
```

Comment:
This isn't how property docstrings are formatted. See other properties for details